zmscitizenapi/tests/Zmscitizenapi/Controllers/System/SourceCacheWarmupControllerTest.php

```
@@ -0,0 +1,151 @@
+class SourceCacheWarmupControllerTest extends ControllerTestCase
+{
+    protected $classname = "\BO\Zmscitizenapi\Controllers\System\SourceCacheWarmupController";
+
+    private ?string $previousWarmupToken = null;
+
+    private bool $hadWarmupToken = false;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        \App::$source_name = 'unittest';
```

PHPStan (2.2.8)

[error] Access to an undefined static property App::$source_name.


(staticProperty.notFound)